src/css/header.css

```diff
@@ -1,4 +1,162 @@
-.header {
-  padding: 20px 0;
+.header-section {
+  margin-bottom: 8px;
 }
 
+.header-container {
```

Comment:
Please create separate rule for container in base.css file and use in this place